src/app/modules/layout/home/dashboard/dashboard.component.html

```diff
@@ -1,4 +1,4 @@
-<h2 *ngIf="user?.role === 'learner' || user?.role === 'instructor'">Katalog kurseva</h2>
+<h2 *ngIf="user?.role.includes('learner') || user?.role === 'instructor'">Katalog kurseva</h2>
```

Comment:
Do we need to use "includes" for the instructor here as well?

Reply:
No, instructor has only one role.